waspc/packages/deploy/src/providers/fly/index.ts

```diff
@@ -41,6 +41,10 @@ class FlyCommand extends Command {
         "--volume-size <volumeSize>",
         "flyctl postgres create option",
         "1",
+      )
+      .option(
+        "--db-image <dbImage>",
+        "custom postgres image with extensions (must be based on flyio/postgres-flex)",
```

Comment:
I would make it clear it is Docker image.
Also, why "with extensions"? Does it need to have extensions, why would that be so?